File: BECore/app/model/ProjectPlanExpense_AttachDetail.js
/// <summary>
/// Model EXTJS ProjectPlanExpense_AttachDetail.
/// </summary>
Ext.define('QLDT.model.ProjectPlanExpense_AttachDetail', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectPlanExpense_AttachDetailID',
    idType: 'Guid',
    fields: [	
        { name: "ProjectPlanExpense_AttachDetailID", type: "string" },
        { name: "ProjectPlanExpenseID", type: "string" },
		{ name:"FileName", type:"string" },
		{ name: "FileType", type: "string" },
        { name: "Description", type: "string" },
		{ name:"FileSize", type:"string" },
		{ name:"Inactive", type:"boolean" },
		{ name:"SortOrder", type:"int" },
		{ name:"CreatedDate", type:"date" },
		{ name:"ModifiedDate", type:"date" },
		{ name:"IPAddress", type:"string" },
		{ name:"ModifiedBy", type:"string" },
		{ name: "CreatedBy", type: "string" },
        { name: "IsTemp", type: "boolean" },
     ]
});

File: MT.Model/Business/ProjectPlanExpense_AttachDetail.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectPlanExpense_AttachDetail:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectPlanExpense_AttachDetail:EntityBase.
    /// </summary>
    public class ProjectPlanExpense_AttachDetail : EntityBase
    {
        [Key]
        public Guid ProjectPlanExpense_AttachDetailID { get; set; }
        public Guid ProjectPlanExpenseID { get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public double FileSize { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
        public Guid? FileResourceID { get; set; }
        public string Description { get; set; }
    }
    #endregion
}

